### PR TITLE
PSP-6475 : Generate Expropriation Form - Form 8

### PR DESCRIPTION
**Pull request by eddherrera**
image

image

### CODE REVIEW

File: source/frontend/src/models/generate/GenerateAddress.ts [hunk 1]
```diff
@@ -43,8 +43,9 @@ export class Api_GenerateAddress {
       .filter(a => !!a)
       .join(', ');
 
-    this.address_String_multiline_compressed =
```
Comment (devinleighsmith) [inline source/frontend/src/models/generate/GenerateAddress.ts:46]: @eddherrera is this used in any other templates? If so does this require us to update those templates?

Comment (eddherrera) [reply source/frontend/src/models/generate/GenerateAddress.ts:46]: no, this is the only template so far that uses this format. I created this variation of the address so it's new.

**Comment by devinleighsmith** [comment]
This should have been part of the previous pr that introduced this but it also affects this pr.
If an owner_string is an org, I'm guessing that Inc. No should only be included if inc. no is non-null:

Current logic:
    generateOwner.owner_string = `${generateOwner.last_name_or_corp_name}, Inc. No. ${generateOwner.incorporation_number}`;

@eddherrera

**Review by devinleighsmith — APPROVED** [review]

**Comment by devinleighsmith** [comment]
This is fine actually, you can fix this with your next pr.

**Pull request merged**